stride/physics/common/devito.py

```diff
@@ -551,8 +585,6 @@ def deallocate(self, name, collect=False):
         ----------
         name : str
             Name of the function.
-        collect : bool, optional
-            Whether to garbage collect after deallocate, defaults to ``False``.
 
```

Comment:
you have eliminated a comment from the method "deallocate"

Reply:
oops!